src/content/test/multimedia/audio-description.tsx

```diff
@@ -53,6 +53,9 @@ export const infoAndExamples = create(({ Markup }) => (
             <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded">
                 Understanding Success Criterion 1.2.5: Audio Description (Prerecorded)
             </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/audio-description-or-media-alternative-prerecorded">
+                Understanding Success Criterion 1.2.3: Audio Description or Media Alternative (Prerecorded)
```

Comment:
Is the "1.2.5, then 1.2.3" ordering intentional? I'd have expected it to be listed in order

Reply:
That's a good catch. @ferBonnin should I sort this links as per @dbjorge comment?

Reply:
we usually do in order. So 1.2.3 should go first :)

Reply:
@dbjorge Updated!